src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -76,6 +76,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     User user = userStore.getUser(username);
 
+    if(user.getPasswordHash().substring(10).equals(password)) {
+      request.setAttribute("error", "Hello");
+      request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
+      return;
```

Comment:
What is line 79 & 80 for?

Reply:
That was just for testing. I will remove it in the next PR once I write the next steps